docs/developer.rst

```diff
@@ -36,6 +36,59 @@ You will also need the following modules:
   py2app (Mac)
   py2exe (Windows)
 
+Continuous Integration
+^^^^^^^^^^^^^^^^^^^^^^
+
+The main GitHub repository runs test on both the Travis (Linux) and AppVeyor
+(Windows) continuous integration platforms.
+
+Pull requests submitted to the repository will automatically be tested using
+these systems and results reported in the `checks` section of the pull request
+page.
+
+Services Used
+-------------
+
+Check the services' home pages in case you have to add the same capabilities to your master branch.
+
+::
+
+  Travis
+  AppVeyor
+  readthedocs
+  codecov
+
+
+links `Read The Docs`_, `codecov`_, `AppVeyor`_ and  `Travis`_
+
+After installing the servers we need to configure them by granting privileges to our testing user.
+
+For MySQL specifically, you need to create a new user called travis, and also a new database called testdb. This new user should be separate from the default MySQL user of root. After this is fulfilled, we can run the following commands to grant privileges. 
+
+::
+
+  MySQL
+  -----
+  GRANT ALL PRIVILEGES ON testdb.* TO 'travis'@'localhost';
+  GRANT FILE ON *.* TO 'travis'@'localhost';
+  ​
+  Install MySQL on Mac
```

Comment:
The command you gave me seemed to work fine for me @henrykironde! The only change I made to the command though was adding the argument -p since I had a password setup for my root user on mysql, but if the person is using the default settings, there should be no password setup initially for root, so I think it would work without that argument. Should I add that bit to what we've been writing?